devel/bazel/Portfile

```diff
@@ -29,115 +29,66 @@ set extra_args ""
 # Note bazel-3.{1,5,7} build on OSX10.11 but fail at runtime with
 # dyld: lazy symbol binding failed: Symbol not found: _os_log_create
 
+set min_darwin      16
+
 if { ${name} eq ${subport} } {
 
     # Main port
-    github.setup      bazelbuild ${name} 4.2.2
-    revision          0
+    github.setup    bazelbuild ${name} 6.1.1
+    revision        0
 
-    checksums         rmd160  9616833d5b2be2181bdb4dc05d8064d89017051d \
-                      sha256  9981d0d53a356c4e87962847750a97c9e8054e460854748006c80f0d7e2b2d33 \
-                      size    279304631
+    checksums       rmd160  821f9b2c6bdca19feb405d52e8823e7fc987c731 \
+                    sha256  6b900f26d676c7eca1d2e7dff9b71890dabd3ff59cab2a2d2178bc8a0395342a \
+                    size    194630986
 
     set bazel_min_xcode 12.0
     compiler.blacklist-append {clang < 1200}
     compiler.blacklist-append {macports-clang-[5-9].0}
-    set min_darwin    16
-
 } elseif { "bazel-3.7" eq ${subport} } {
+    PortGroup       obsolete 1.0
 
     # Specific version for py-tensorflow - DO NOT CHANGE
-    github.setup      bazelbuild ${name} 3.7.2
-    revision          1
-
-    # livecheck none is correct here, see above.
-    livecheck.type    none
-
-    checksums         rmd160  abf23b277194854bbd4e4390ab6b3f971d26826c \
-                      sha256  de255bb42163a915312df9f4b86e5b874b46d9e8d4b72604b5123c3a845ed9b1 \
-                      size    313677756
-
-    compiler.blacklist-append {clang < 710}
-    compiler.blacklist-append {macports-clang-[7-9].0}
-    set min_darwin    16
+    github.setup    bazelbuild ${name} 3.7.2
+    revision        1
 
+    # Obsolete Date: 2023-08-01
+    replaced_by     bazel
```

Comment:
`replaced_by` doesn't work unless you increase `revision` or `version` or `epoch`.

Obsoleting these ports will break all the ports that depend on them, for example py-dm-tree.

Reply:
`revision` bumped.